iothub_client/samples/iothub_client_sample_upload_to_blob_custom/iothub_client_sample_upload_to_blob_custom.c

```diff
@@ -0,0 +1,197 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Filename could be more descriptive: it's hard to understand what "custom" means.
Maybe `_with_retry` or `_get_sas`?